include/libopencm3/stm32/common/ltdc_common_f47_l4.h

```diff
@@ -7,8 +7,8 @@
  *
  * @date 5 December 2014
  *
- * This library supports the LCD TFT display controller (LTDC) in the STM32F4xx
- * and STM32F7xx series of ARM Cortex Microcontrollers by ST Microelectronics.
+ * This library supports the LCD TFT display controller (LTDC) in the STM32F4xx,
+ * STM32F7xx, and STM32L4xx series of ARM Cortex Microcontrollers by ST Microelectronics.
```

Comment:
If you want reviewers to have sane chance of reviewing this, please break this up into smaller steps. You're trying to add L4+.  This is fixing comments in a file that's already used by L4 and already wrong, and will also be used by L4+.  It's _correct_ to fix it, don't get me wrong, but it's got to be in separate commits (see also my comment about renaming files)

Reply:
I've started splitting it into several commits at the same time I rebased on the last master branch to fix the merge conflict.